<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GUI_Broadcast</class>
 <widget class="QWidget" name="GUI_Broadcast">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>364</width>
    <height>67</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="verticalSpacing">
    <number>5</number>
   </property>
   <item row="2" column="0" colspan="2">
    <widget class="Gui::ComboBox" name="combo_clients">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="focusPolicy">
      <enum>Qt::ClickFocus</enum>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="Gui::MenuToolButton" name="btn_menu">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string notr="true">â‰¡</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="3">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="lab_status">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="focusPolicy">
        <enum>Qt::ClickFocus</enum>
       </property>
       <property name="lineWidth">
        <number>0</number>
       </property>
       <property name="text">
        <string notr="true">3 users listening</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lab_error">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Cannot Broadcast</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_retry">
       <property name="text">
        <string notr="true">Retry</string>
       </property>
       <property name="icon">
        <iconset resource="../../Resources/Icons.qrc">
         <normaloff>:/Icons/undo.svg.png</normaloff>:/Icons/undo.svg.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>16</width>
         <height>16</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Gui::ComboBox</class>
   <extends>QComboBox</extends>
   <header>Gui/Utils/Widgets/ComboBox.h</header>
  </customwidget>
  <customwidget>
   <class>Gui::MenuToolButton</class>
   <extends>QPushButton</extends>
   <header location="global">Gui/Utils/MenuTool/MenuTool.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../Resources/Icons.qrc"/>
 </resources>
 <connections/>
</ui>
